### PR TITLE
#7507 full screen button 

### LINKED ISSUE #7507: Full screen button
image
https://canary.kodadot.xyz/ahp/gallery/13-1003

- the image should fit on the page
- also, please make the preview bg full height minus navbar
image

- the gaps there should not be there 
- would also be nice if the go back button would not cover the img

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #7507
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [x] My fix has changed UI
Screen Shot 2023-10-04 at 9 59 04 PM

### CODE REVIEW

**Comment by prachi00** [comment]
@roiLeo 
so about 
can't we display it over the navbar?

@exezbcz mentioned preview minus navbar in the ticket 

Screen Shot 2023-10-11 at 10 22 47 AM

File: components/gallery/GalleryItemPreviewer.vue [hunk 1]
```diff
@@ -50,6 +50,7 @@ const isFullscreen = useVModel(props, 'value', emit)
 
 .gallery-item-modal {
   position: fixed;
+  margin-top: 5.25rem !important;
 
   :deep(&-content) {
```
Comment (preschian) [inline components/gallery/GalleryItemPreviewer.vue:55]: @prachi00 maybe change this line to this help fix the issue?

```suggestion
  :deep(.gallery-item-modal) {
```

Comment (prachi00) [reply components/gallery/GalleryItemPreviewer.vue]: you are right, thanks

**Review by exezbcz — APPROVED** [review]

**Comment by yangwao** [comment]
What's about making fullscreen as tap-on picture? It make more apple-like sense to me rather have aim into a corner?
- https://areebah.art/character-illustrations
https://github.com/kodadot/nft-gallery/assets/5887929/eb0d883d-5c08-4710-b07f-896b29db6bba

**Comment by yangwao** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 3.8 USD/DOT ~ 2.632 $DOT
🧗 [13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1](https://polkadot.subscan.io/account/13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)
🔗 [0x34e9a6f5ec8541052bb19736a438fa1d2e616d4de36d2a6956784a5867c5e8af](https://polkadot.subscan.io/extrinsic/0x34e9a6f5ec8541052bb19736a438fa1d2e616d4de36d2a6956784a5867c5e8af)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**